spec/components/admin/npq/applications/analysis/table_row_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe Admin::NPQ::Applications::Analysis::TableRow, type: :view_component do
```

Comment:
This will need to be `:component` to pull in `render_inline` and friends (although they're not being used yet)

```suggestion
RSpec.describe Admin::NPQ::Applications::Analysis::TableRow, type: :component do
```